### DISCUSSION TITLE
How do I embed various documents (LaTeX and Typst) as HTML webpages in Quarto?

### DISCUSSION THREAD

**Discussion by alex180500** [quarto-dev/quarto-cli#10121] [Q&A] [answered]
### Description

Hello! I'm starting out with quarto, I have multiple documents, some papers I've written with various journal templates in LaTeX and a thesis which I'm writing in Typst.

I'm creating a website with a blog. **Ideally I would like to add one page of the blog that takes the pure LaTeX files (or the Typst files) and renders them as a HTML webpage.** Ideally without changing much of them.

How can I do this?

**Comment by cderv** [comment]
> Ideally I would like to add one page of the blog that takes the pure LaTeX files (or the Typst files) and renders them as a HTML webpage

It is not clear to me what is your input. Do you want to show in the website the rendered PDF from .qmd rendering through LaTeX or Typst ? Or do you have LaTeX and Typst file as input, and you want to show the rendered PDF ? 

Can you say more on this ?

**Reply by alex180500** [reply]
Yes sorry if I was unclear. I have a LaTeX file as input and I want to create an html webpage (similar to the old arxiv vanity)

**Reply by mcanouil** [reply]
Quarto main input is Markdown, but you can use Pandoc to perform conversion (with possible loss).

**Reply by alex180500** [reply]
I thought there was a simpler way like making a qmd file and like "importing" the latex source, or something similar.

I can for example use ipynb files as webpages, so I thought there was a similar way for latex files but maybe I am just wrong

**Reply by mcanouil** [reply, marked as answer]
Jupyter has Markdown and code cells, so still markdown.
Here you want to include other languages.

Something has to do the job to translate Typst/LaTeX to HTML, and that's what Pandoc does.
Even with Pandoc, it's not a "throw anything it will guess and handle it". You have to tell it what is what and to what you want Pandoc to translate.

Quarto is using Pandoc but as said, it's role is not to convert anything to anything. It's to allow from Markdown to get access to multiple formats.
